### PR TITLE
ADD Maxwell assign radiation

### LINKED ISSUE #3764: Add assign radiation boudary in Maxwell
### Description of the feature

in HFSS, there are assign radiation settings like bellow

https://aedt.docs.pyansys.com/version/stable/API/_autosummary/pyaedt.hfss.Hfss.assign_radiation_boundary_to_faces.html
https://aedt.docs.pyansys.com/version/stable/API/_autosummary/pyaedt.hfss.Hfss.assign_radiation_boundary_to_objects.html

but it appears that Maxwell 2D and 3D do not have radiation boudary setting

image

Can we implemented this function? or Is there any alternative function?

### Steps for implementing the feature

oModule = oDesign.GetModule("BoundarySetup")
oModule.AssignRadiation(
	[
		"NAME:Radiation1",
		"Objects:="		, ["Region"]
	])

### Useful links and references

_No response_

### PR DESCRIPTION
**Pull request by Schwalbe262**
ADD Maxwell assign radiation

oModule = oDesign.GetModule("BoundarySetup")
oModule.AssignRadiation(
[
"NAME:Radiation1",
"Objects:=" , ["Region"]
])

Close #3764

### CODE REVIEW

**Comment by Samuelopez-ansys** [comment]
@Schwalbe262 Please add the unit tests corresponding to this new method. Thanks

**Comment by Samuelopez-ansys** [comment]
@Schwalbe262 Please add the corresponding docstring at the beginning of the method, it will be used to create automatically the documentation, please check other methods if you need help.

**Comment by gmalinve** [comment]
@Samuelopez-ansys I'll just add another assert:
`
bound3 = self.aedtapp.assign_radiation([rect, rect2, box, box2.faces[0]], "my_rad")
assert bound2.name != bound3.name
`

but it's up to you :)

**Review by gmalinve — APPROVED** [review]

File: _unittest/test_28_Maxwell3D.py [hunk 1]
```diff
@@ -916,3 +916,16 @@ def test_56_zero_tangential_h_field(self, add_app):
...
+    def test_34_radiation(self):
+        self.aedtapp.insert_design("Radiation")
+        rect = self.aedtapp.modeler.create_rectangle(0, [0, 0, 0], [5, 5], matname="aluminum")
+        rect2 = self.aedtapp.modeler.create_rectangle(0, [15, 20, 0], [5, 5], matname="aluminum")
+        box = self.aedtapp.modeler.create_box([15, 20, 0], [5, 5, 5], matname="aluminum")
+        box2 = self.aedtapp.modeler.create_box([150, 20, 0], [50, 5, 10], matname="aluminum")
+        bound = self.aedtapp.assign_radiation([rect, rect2, box, box2.faces[0]])
+        assert bound
+        bound2 = self.aedtapp.assign_radiation([rect, rect2, box, box2.faces[0]], "my_rad")
+        assert bound2
```
Comment (Samuelopez-ansys) [inline _unittest/test_28_Maxwell3D.py:929]:
```suggestion
        assert bound2
        bound3 = self.aedtapp.assign_radiation([rect, rect2, box, box2.faces[0]], "my_rad")
        assert bound2.name != bound3.name
```

**Review by Samuelopez-ansys — APPROVED** [review]

**Pull request merged**